docs/site-example/packages

```diff
@@ -0,0 +1,17 @@
+default({
+	Package('iwinfo'),
+	Feature('autoupdater'),
+	Feature('ebtables-filter-multicast'),
+	Feature('ebtables-filter-ra-dhcp'),
+	Feature('ebtables-limit-arp'),
+	Feature('mesh-batman-adv-15'),
+	Feature('mesh-vpn-fastd'),
+	Feature('respondd'),
+	Feature('status-page'),
+	Feature('web-advanced'),
+	Feature('web-wizard')
```

Comment:
How about making this:
```
Features {
	'autoupdater',
	'ebtables-filter-multicast',
	'ebtables-filter-ra-dhcp',
	'ebtables-limit-arp',
	'mesh-batman-adv-15',
	'mesh-vpn-fastd',
	'respondd',
	'status-page',
	'web-advanced',
	'web-wizard',
}
```
(as Lua conveniently allows us to omit the `()` when there is a single table argument)

I'm also wondering if we really need the `default` and `when` functions - maybe something like a toplevel `features`/`packages` function and `if not device_class('tiny')` would also work? I have not really through about the the implementation though.

Finally, is it possible to remove features or packages with this syntax (as supported in `site.mk` with a `-` prefix)?